src/server/routes/suppliers.js

```diff
@@ -73,9 +73,12 @@ let createSuppliers = function(req, res)
 
 let sendSupplier = function(req, res)
 {
-  Supplier.find(req.params.supplierId).then(suppliers =>
+  const queryObj = req.params.supplierId ? { supplierId: req.params.supplierId } : {};
+  const includes = req.query.include ? req.query.include.split(',') : [];
+
+  Supplier.all(queryObj, includes).then((supplier) =>
```

Comment:
This returns an array and not an object. I expect the `Supplier.find` method to be modified